doc/source/rayserve/overview.rst

```diff
@@ -192,6 +192,24 @@ To scale out a backend to multiple workers, simplify configure the number of rep
 
 This will scale out the number of workers that can accept requests.
 
+Using Resources (CPUs, GPUs)
+++++++++++++++++++++++++++++
+To assign hardware resource per worker, you can pass resource requirements to
+``ray_actor_options``. To learn about options to pass in, take a look at
+:ref:`Resources with Actor<actor-resource-guide>` guide.
+
```

Comment:
Add a sentence here describing the example so it's easier to scan:

`For example, to create a backend where each replica uses a single GPU, do the following:`